tokio/src/macros/select.rs

```diff
@@ -429,7 +429,8 @@ macro_rules! select {
         //
         // This module is defined within a scope and should not leak out of this
         // macro.
-        mod util {
+        #[doc(hidden)]
+        mod __tokio_util {
```

Comment:
take it or leave it: this is specifically for the `select` macro, so i might call it `__tokio_select_util`? not a big deal either way though.

Reply:
Agreed - I've reset my PR to the current master with the doc fix you made to reduce clutter and made the requested change.